src/components/button-minor/button-minor.stories.mdx

```diff
@@ -0,0 +1,417 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs";
```

Comment:
Recently we've switched to having the user facing stories in a separate TS files (to have an additional layer of testing the prop interfaces we provide) which are imported and embedded in `mdx` files like this. We're doing this because typechecking, eslint and intellisense unfortunately does not work in `mdx` files. It would be great if you could refactor this file and move the stories to a new`button-minor.stories.tsx` file.

You can check a `Card` component and `card.stories.mdx` and `card.stories.tsx` files as an example of how we're doing it right now.

Reply:
I think it would be great to raise a separate PR for this task.